### ISSUE TITLE
[Workbench] Can't create "related to" relationships and therefore relation with Observables

### ISSUE THREAD

**Issue by Lhorus6** [LuatixHQ/opencti#7908]
## Description

We are currently stuck on fully modeling Incidents because it is impossible to make relationships between an Incident and an Observable.

These relationships are of the "related to" type but the workbench does not allow to make this type of relationship.

Impact : impacts future detection engineering and research efforts

## Environment

OCTI 6.2.10

## Reproducible Steps

Steps to create the smallest reproducible scenario:
1. Create a workbench
2. Create an Incident
3. Try to create a relationship between your Incident and an Observables
-> You can't create "related to" relation, so you can't create relation with an Observables

![Screenshot 2024-08-01 094724](https://github.com/user-attachments/assets/9ccf312b-1e2d-4331-97aa-a5c46eaf7bf3)

**Comment by nino-filigran** [comment]
all the relations in workbench do not exist: we'll update this one, but that exactly the reason why we would like to have drafts instead of workbenches. We'll be able to leverage all possible relations directly within drafts. Given the related to relation is not present for any entity, we'll add it only for incident, to limit dev effort for now.

**Comment by Lhorus6** [comment]
Perfect, thanks

**Comment by SarahBocognano** [comment]
For now in the platform the relations for an SDO and an Observable are not manage in the workbench. 
After a investigation in the code base ,adding a "related-to" between an SDO and an Observable will required a significant amount of work. Not sure that is worth the time considering the Draft is coming and accepting relationships between those two.

**Comment by nino-filigran** [comment]
Agreed @SarahBocognano, I still leave this bug open for information, but will decrease the priority.

**Comment by Lhorus6** [comment]
On my side, it doesn't work on demo 🤔 

Let's have a look together

**Comment by JeremyCloarec** [comment]
I'm closing this issue because of the future deprecation of workbenches and the availability of drafts.
If you still consider it a high priority fix @Lhorus6, feel free to re-open it
